src/pages/components/footerSection/index.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import DownloadButton from "../DownloadButton";
+import FooterContent from "./FooterContent";
+
+const FooterSection = () => {
+  return (
+    <div className="bg-main-backgound">
```

Comment:
remove the background, instead make use of this https://github.com/framgia/sph_ui-ux-swipet/pull/6/files#r861418188